app/javascript/packs/component-definition-common.js

```diff
@@ -0,0 +1,5 @@
+import CustomMonitoringForm from '../components/custom_monitoring_form';
+
+ManageIQ.component.addReact(
+  "CustomMonitoringForm", CustomMonitoringForm
```

Comment:
this is weird, only 2 params can be passed to addReact, why not keep it on 1 line?